example/spec/serial/AsyncOperationRunnerFailingLambdaSpec.js

```diff
@@ -49,7 +49,7 @@ describe('The AsyncOperation task runner executing a failing lambda function', (
         id: asyncOperationId,
         taskArn: randomString(),
         description: 'Some description',
-        operationType: 'ES Index',
+        operationType: 'Bulk Granules',
```

Comment:
Just for clarity? Seems great, just curious.

Reply:
I'm sorry, I'm not sure I understand what you're asking here.

Reply:
yeah I got the types from /packages/types/api/async_operations. I switched it since we shouldn't be doing (most of) the ES things anymore. I considered removing the type, but Jonathan pointed out that past operations might still have the type. Not sure how long that will be for though.